components/quiz/answeredQuestionToShow.js

```diff
@@ -0,0 +1,55 @@
+import {get} from 'lodash'
+import {useMachine} from '@xstate/react'
+import {quizMachine} from 'machines/quizMachine'
+
+import {
+  MultipleChoice,
+  Essay,
+  Theater,
+  Sketch,
+  MultipleImageChoice,
+} from 'components/quiz'
+
+export default function AnsweredQuestionToShow({
+  question,
+  answers,
+  quizId,
+  ...props
+}) {
+  const [state, send] = useMachine(quizMachine, {
+    context: {
+      quizId: quizId,
+      // Passing answers from main machine causes
+      // this machine to skip initializing (loading).
+      // Answers are just questions with answer value.
+      questions: answers,
+    },
+  })
```

Comment:
Invoking new machine for answered questions and skipping its initialization (to avoid fetching same data twice) by passing array of answers from the parent machine. answered questions live in their own context.